app/assets/stylesheets/pages/policy.scss

```diff
@@ -0,0 +1,133 @@
+.policy-page  {
+  margin-top: 60px;
+  max-width: 1024px;
+  margin-bottom: 48px;
+
+  .policy-intro {
+    &__title {
+      font-size: 18px;
+      height: 24px;
+      font-weight: bold;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: 1.33;
+      letter-spacing: 0.3px;
+      color: $text-title;
+      margin-left: 6.5%;
+    }
+    &__paragraph {
+     margin-top: 16px;
+     height: 120px;
+     font-size: 18px;
+     font-weight: normal;
+     font-style: normal;
+     font-stretch: normal;
+     line-height: 1.33;
+     letter-spacing: normal;
+     color: $text-body;
+     letter-spacing: 0.13px;
+     margin-left: 6.5%;
+     margin-bottom: 20px;
+   }
+   &__line {
+     width: 452px;
+     height: 1px;
+     margin-left: 6.5%;
+     color: #bbbbbb;
+     background-color: #bbbbbb;
+     margin-top: .25%;
+   }
+  }
+
+  .disclosures {
+    margin-top: .5%;
+    margin-left: 5%;
+    &__faq {
+      width: 192px;
+      height: 24px;
+      font-size: 18px;
+      font-weight: bold;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: 1.33;
+      letter-spacing: 1px;
+      color: $text-subtitle;
+      margin-left: 1%;
+    }
+    &__information {
+      padding-left: 80px;
+      h1 {
```

Comment:
@emilyspieler A heads up we'll want to refactor these out to a main style guide that had global styles like these types of titles defined. I think @gcwokeup is working on it.